test/local/db.js

```diff
@@ -336,7 +346,7 @@ describe('redis enabled, token-pruning enabled:', () => {
   })
 
   it('should call redis.update in db.deleteSessionToken', () => {
-    return db.deleteSessionToken({ id: 'wibble', uid: 'blee' }, P.resolve())
+    return db.deleteSessionToken({ id: 'wibble', uid: 'blee' })
```

Comment:
The trailing `P.resolve()` was old copy-past, removed here and a few places further down.